code/modules/busy_space/organizations.dm

```diff
@@ -31,6 +38,7 @@
 		"Explorer",
 		"Swift",
 		"Dragonfly",
+		"Hollyhock",
```

Comment:
what does it mean.....

Reply:
flower of ambition. but also I had to add something to cause a change so I could try and fix some git mistakes :tm: I made

Reply:
oh yeah I meant entirely in a "is this a top secret in-joke" because it was so... singular

Reply:
oh i mean i submitted it as a name for the calvera ages ago but beyond that no its just a word i think would make a good spaceship name